.snyk

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
This file will need to make it to `main` before Snyk can be set up to scan `requirements.txt` and `tests/test_requirements.txt`. Snyk defaults to Python 3.9, and those requirements include things that need a later version of Python to be able to be installed.